### PR TITLE
⭐General Code Cleanup & Overall Greatness

### PR DESCRIPTION
**Pull request by NotGeri**
[[`kommit 69420`](https://geri.dev/commit-691)]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (KoxSosen) [inline index.html:1]: This change introdues a possible exploit which can be found, and exploited in chrome browsers, as mentioned [here](https://www.cvedetails.com/cve/CVE-2019-13735/). 

As an additon, please see [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ) why this is actually a breaking change.

**Review by KoxSosen — COMMENTED** [review]
Fix dis

File: index.html (hunk 1 above)
Comment (NotGeri) [reply index.html:1]: no fix I hacc

**Comment by KoxSosen** [comment]
Thanks for the fix and the rick roll, love u mate :p

**Comment by NotGeri** [comment]
BAD PROJECT

**Comment by KoxSosen** [comment]
Hahaaaaaaaa!

**Comment by KoxSosen** [comment]
Slow, too bad!

File: index.html (hunk 1 above)
Comment (KoxSosen) [reply index.html:1]: Calling the internet police

**Pull request merged**